src/lib/ibcclient.ts

```diff
@@ -721,6 +752,7 @@ export class IbcClient {
     myConnectionId: string,
     proof: ConnectionHandshakeProof
   ): Promise<MsgResult> {
+    this.logger.info(`Connection open ack for my connection ${myConnectionId}`);
```

Comment:
Yes. Just connections here, not clients (we printed the connectionIDs before.

Let's add both connection IDs: `Connection open ack for ${myConnectionId} => ${proof.connectionId}`